drivers/kw2xrf/include/kw2xrf_params.h

```diff
@@ -46,14 +46,15 @@ extern "C" {
 #define KW2XRF_PARAM_SLEEP       (GPIO_PIN(0, 2))
 #endif
 #ifndef KW2XRF_PARAM_RESET
-#define KW2XRF_PARAM_RESET       (GPIO_PIN(0, 3))
+#define KW2XRF_PARAM_RESET       (GPIO_PIN(1, 19))
```

Comment:
exactly, it was not used. But sure, I can move that

Reply:
I added in the header file that the pin configuration corresponds to the `pba-d-01-kw2x`.
This is similar to what we have in the other devices.

In the mid-term, we should probably define these as no-op and add board specific params. We can also add some common headers (e.g arduino)

Reply:
Did you commit/push the board changes?

Reply:
no, I didn't, since I based on the other devices (and because that would simply duplicate this list to the driver).
Or should I do it anyway?

Reply:
Since the board doesn't define this pin, but relies on the default config instead, it's configuration would now change and presumably be no longer valid.